packages/table/src/table2.tsx

```diff
@@ -430,6 +430,42 @@ export class Table2 extends AbstractComponent2<Table2Props, TableState, TableSna
         this.quadrantStackInstance.scrollToPosition(correctedScrollLeft, correctedScrollTop);
     }
 
+    /**
+     * Can adjust the scroll offset the table by the provided x and y values. Can also set a
+     * linear gradient overlay effect to the table to indicate scrolling with the scrolling
+     * direction provided.
+     *
+     * - relativeOffset: The horizontal and vertical scroll modification to be applied.
```

Comment:
use JSDoc to describe parameters

```suggestion
     * @param relativeOffset - how much to scroll the table body in pixels relative to the current scroll offset
```